zproject/settings.py

```diff
@@ -1107,3 +1107,6 @@ def get_secret(key):
 CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
 
 CONTRIBUTORS_DATA = os.path.join(STATIC_ROOT, 'generated/github-contributors.json')
+
+# Controls if emails are sent on logins to an account
+SEND_LOGIN_EMAILS = True
```

Comment:
this should go in DEFAULT_SETTINGS in this file.  Read http://zulip.readthedocs.io/en/latest/settings.html for why :)